### PR TITLE
IA-3189: Updated to support terraform provider ~>5.0

### PR DESCRIPTION
**Pull request by yaythecloud**

### CODE REVIEW

File: versions.tf [hunk 1]
```diff
@@ -3,7 +3,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 4.0"
+      version = "~> 5.0"
```
Comment (bushong1) [inline versions.tf:6]: this is a breaking change, should be a major bump unless you set "=> 4.0, < 6.0"

**Review by mshiyk — DISMISSED** [review]
In CHANGELOG.md added an update comment saying that terraform provider version is now 5.

In versions.tf , assigned 5.0 to the version variable.

File: versions.tf (hunk 1 above)
Comment (yaythecloud) [reply versions.tf:6]: @bushong1 thank you for pointing this out. Configured the CHANGELOG to reflect this with a major version bump.

**Review by mshiyk — APPROVED** [review]
In CHANGELOG.md added an update comment saying that terraform provider version is now 5.

In versions.tf , assigned 5.0 to the version variable.

**Pull request merged**